src/test/fixtures/sample-config.json

```diff
@@ -573,6 +568,11 @@
           }
         }
       },
+      "uiSchema": {
+        "remarks": {
+          "ui:widget": "textarea"
+        }
+      },
```

Comment:
Could you add a case where it's one more level deeper? Just wanted to be certain it works for nested key-values pairs

Reply:
done, created two cases:
- one where `uiSchema` is at the form template's root
- another where `uiSchema` is nested within `defaults` and within `defaults.misc`